storage/core/src/main/java/com/hortonworks/registries/storage/impl/jdbc/provider/oracle/query/OracleUpdateQuery.java

```diff
@@ -62,8 +63,10 @@ protected String createParameterizedSql() {
     }
 
     private Map<Schema.Field, Object> createWhereClauseColumnToValueMap() {
-        Map<Schema.Field, Object> bindingMap = getBindings().stream().
-                collect(Collectors.toMap(keyValuePair -> keyValuePair.getKey(), keyValuePair -> keyValuePair.getValue()));
+        Map<Schema.Field, Object> bindingMap = new HashMap<>();
```

Comment:
@raju-saravanan confirmed offline that earlier `stream().collect(Collectors.toMap)` operation throws an error if there are duplicate keys.